test/components/author-link.test.tsx

```diff
@@ -1,17 +1,19 @@
 import React from 'react';
 import renderer from 'react-test-renderer';
-import AuthorLink from '../../src/components/author-link';
+import Author from '../../src/components/author';
 
 describe('AuthorLink component', () => {
```

Comment:
nit: Since this file now tests the `Author` component instead of `AuthorLink`, do we want to rename it to `author.test.tsx`?

If we agree, its also a good idea to update the above block to 
```suggestion
describe('Author component', () => {
```

Reply:
I'd leave that up to you for another PR :) 👍